core/src/main/java/org/jboss/hal/core/runtime/TopologyTasks.java

```diff
@@ -95,7 +95,7 @@ public final class TopologyTasks {
     public static final String HOSTS = "topologyTasks.hosts"; // List<Host>
     public static final String SERVER_GROUPS = "topologyTasks.serverGroups"; // List<ServerGroup>
     public static final String SERVERS = "topologyTasks.servers"; // List<Server>
-    public static final String SERVER = "topologyTasks.server"; // Server
```

Comment:
Please leave the constant unchanged.

Reply:
And change the code in line [664](https://github.com/hal/console/blob/main/core/src/main/java/org/jboss/hal/core/runtime/TopologyTasks.java#L664) from:

```java
ResourceAddress address = new ResourceAddress()
        .add(ModelDescriptionConstants.HOST, host)
        .add(SERVER, WILDCARD);
``` 

to:

```
ResourceAddress address = new ResourceAddress()
        .add(ModelDescriptionConstants.HOST, host)
        .add(ModelDescriptionConstants.SERVER, WILDCARD);
```